Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -322,6 +331,37 @@ def module_test(client: Client):
     demisto.results('ok')
 
 
+def fetch_indicators(client: Client, auto_sync: bool = False):
+    if demisto.getIntegrationContext() and auto_sync:
+        xdr_iocs_sync_command(client, first_time=True)
+    else:
+        get_changes(client)
+        if auto_sync:
+            tim_insert_jsons(client)
+            if iocs_to_keep_time():
+                xdr_iocs_sync_command(client)
+
+
+def xdr_iocs_sync_command(client: Client, first_time: bool = False):
+    if first_time:
```

Comment:
if the integration context is empty, then you need to run `sync` too, right?
better safe than sorry - don't assume that the actual first call will be with `first_time` true.